runtime/src/main/java/org/corfudb/runtime/collections/StreamPollingTask.java

```diff
@@ -104,7 +100,7 @@ private void pollTxnStream() throws Exception {
         if (updates.isEmpty()) {
             log.trace("pollTxStream :: no updates for {} from {}, listenerId={}", txnStream.getId(),
                     lastReadAddress + 1L, subscription.getListener().getClass().getSimpleName());
-            pollingExecutor.schedule(this, randomGenerator.nextInt(pollingIdleWaitTime), TimeUnit.MILLISECONDS);
+            pollingExecutor.schedule(this, pollingIdleWaitTime, TimeUnit.MILLISECONDS);
```

Comment:
should we do some sort of testing to determine what is the best number according to the clients usage pattern?

Reply:
Yes. There are two issues here, the polling strategy and the polling frequency. The current polling strategy aims to lower the load, but it inadvertently doubles it. This PR fixes the first issue to address the second issue.

Reply:
There's a third issue where different streams might want to poll at different periods, but the current APIs forces all pollers to use the same poll period, we can introduce that in subsequent patches if they are necessary.